be/src/common/config.h

```diff
@@ -657,11 +657,11 @@ CONF_Bool(use_hdfs_pread, "true");
 // default: true
 CONF_Bool(rewrite_partial_segment, "true");
 
-// Properties to access aws s3.
-CONF_String(aws_access_key_id, "");
-CONF_String(aws_secret_access_key, "");
-CONF_String(aws_s3_endpoint, "");
-CONF_Int64(aws_s3_max_connection, "102400");
+// Properties to access object storage
+CONF_String(object_storage_access_key_id, "");
```

Comment:
remember to update https://github.com/StarRocks/docs.zh-cn

Reply:
yes, i will update docs